pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -574,6 +606,23 @@ func (ds *Plugin) GetLatestRegistrationEntryEventID(ctx context.Context) (eventI
 	return eventID, nil
 }
 
+func (ds *Plugin) CreateRegistrationEntryEvent(ctx context.Context, event *datastore.RegistrationEntryEvent) (*datastore.RegistrationEntryEvent, error) {
+	return event, ds.withWriteTx(ctx, func(tx *gorm.DB) (err error) {
+		return createRegistrationEntryEvent(tx, event)
+	})
+}
+
+func (ds *Plugin) DeleteRegistrationEntryEvent(ctx context.Context, eventID uint) error {
+	return ds.withWriteTx(ctx, func(tx *gorm.DB) (err error) {
+		return deleteRegistrationEntryEvent(tx, eventID)
+	})
+}
```

Comment:
The only system that deletes the entry events is the SPIRE Server.  They are purged at a long interval, to ensure that all SPIRE Servers have an opportunity to see the event (a requirement for keeping the servers in sync).

Reply:
it si used only for testing, as suggested above if we rename this to `DeleteRegistrationEntryEventForTesting`, we can avoid possible confusions, in case someone want to manage Events in a different way

Reply:
@MarcosDY for unit test usage only